Assets/Scripts/Tutorial/ShelterTutorial.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using k = PPGlobal;
+
+public class ShelterTutorial : Tutorial
```

Comment:
Any notes about living room tutorial apply (using dicts instead of verbose ref list or switch statement)